src/megameklab/com/ui/Aero/tabs/StructureTab.java

```diff
@@ -443,15 +437,12 @@ public void maximizeArmor() {
     
     @Override
     public void useRemainingTonnageArmor() {
-        double currentTonnage = UnitUtil.getEntityVerifier(getAero())
-                .calculateWeight();
+        double currentTonnage = UnitUtil.getEntityVerifier(getAero()).calculateWeight();
         currentTonnage += UnitUtil.getUnallocatedAmmoTonnage(getAero());
         double totalTonnage = getAero().getWeight();
-        double remainingTonnage = TestEntity.floor(
-                totalTonnage - currentTonnage, TestEntity.Ceil.HALFTON);
+        double remainingTonnage = TestEntity.floor(totalTonnage - currentTonnage, TestEntity.Ceil.HALFTON);
         
-        double maxArmor = Math.min(getAero().getArmorWeight() + remainingTonnage,
-                UnitUtil.getMaximumArmorTonnage(getAero()));
+        double maxArmor = Math.min(getAero().getArmorWeight() + remainingTonnage, UnitUtil.getMaximumArmorTonnage(getAero()));
```

Comment:
This line is too long. Per the Google Java Style Guide, which our style guide is based on, columns should be limited to 100 characters. I wouldn't apply it so strictly that I would break up strings, but longer lines require more eye movement and are harder for people like me who work on older displays that can't fit the whole line on the screen at once.

Reply:
I set up a line that is at 100 characters on my IDE, so I can view it. I just missed it in this case. Reverted, and my apologies.

Found out why, my laptop is set to 100 while my desktop is at 120. Fixed the line on my desktop so this doesn't happen again.